app/component/stop-cards/stop-card-list.cjsx

```diff
@@ -12,13 +12,14 @@ class StopCardList extends React.Component
           {@props.children}
         </MasonryComponent>
       </div>
-      <div className="row">
-        <div className="small-10 small-offset-1 medium-6 medium-offset-3 columns">
-          <button className="show-more cursor-pointer" onClick=@props.addStops>
-            <FormattedMessage id="show-more" defaultMessage="Show more" />
-          </button>
-        </div>
-      </div>
+      {if @props.stops && @props.stops.stopsByRadius.pageInfo.hasNextPage
```

Comment:
AddStops is used to increase the max radius, after the stop list has been exhausted for nearest stops, so this should probably have also a toggle in props.